.github/actions/save_logs_and_results/action.yml

```
@@ -0,0 +1,38 @@
+name: save_logs_and_results
```

Why do we not have a human-readable name instead?
```suggestion
name: Save logs and results
```

I assumed quotation marks are not necessary, but I am not so sure about that.